jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListViewImpl.html

```diff
@@ -31,6 +31,12 @@ <h1 data-i18n-key="CaseList"></h1>
                     <div class="form-group">
                         <button class="btn btn-primary" type="button" data-field="start-case" data-i18n-key="StartCase"></button>
                     </div>
+
+                    <div class="form-group pull-right">
+                        <i class="fa fa-refresh"></i>
+                        <button class="btn btn-link" type="button" data-field="refresh-case-list">
```

Comment:
In here, just a minor change, the icon should be inside the button, so no need for the span. We only want to show the refresh icon. Refresh should show as a title for this button when you hover it.
For that to work you need to add `title="..."` to the button and add the following key in the properties bundle: `refresh-case-list-title=Refresh`.